core/server/server_scion.go

```diff
@@ -427,15 +489,15 @@ func StartSCIONServer(ctx context.Context, log *zap.Logger,
 		if err != nil {
 			log.Fatal("failed to listen for packets", zap.Error(err))
 		}
-		go runSCIONServer(ctx, log, mtrcs, conn, localHost.Zone, localHostPort, f)
+		go runSCIONServer(ctx, log, mtrcs, conn, localHost.Zone, localHostPort, f, provider)
```

Comment:
Change `f` to `fetcher` so that old and new code fits better together. (Alternatively, call `provider` just `p` everywhere.)

Reply:
Done